src/test/java/net/sf/jabref/importer/fileformat/BiblioscapeImporterTest.java

```diff
@@ -0,0 +1,76 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.importer.fileformat.BiblioscapeImporter;
+import net.sf.jabref.model.entry.BibtexEntry;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/*
+ * Biblioscape Tag File with an example:
+ * https://web.archive.org/web/20080820010408/http://www.biblioscape.com/manual_bsp/Biblioscape_Tag_File.htm
+ */
+
+public class BiblioscapeImporterTest {
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+    }
+
+    @Test
+    public void testGetFormatName() {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        Assert.assertEquals(importer.getFormatName(), "Biblioscape");
+    }
+
+    @Test
+    public void testGetCLIID() {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        Assert.assertEquals(importer.getCLIId(), "biblioscape");
+    }
+
+    @Test
+    public void testIsRecognizedFormat() throws Throwable {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        List<String> list = Arrays.asList("BiblioscapeImporterTest.txt", "CopacImporterTest1.txt",
+                "CopacImporterTest2.txt", "CopacImporterTest3.txt", "IEEEImport1.txt", "IsiImporterTest1.isi",
+                "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi", "IsiImporterTestMedline.isi",
+                "RisImporterTest1.ris");
+        for (String str : list) {
+            Throwable throwable = null;
+            try (InputStream is = BiblioscapeImporterTest.class.getResourceAsStream(str);) {
+                    Assert.assertTrue(importer.isRecognizedFormat(is));
+            } catch (Throwable var6_8) {
+                if (throwable == null) {
+                    throwable = var6_8;
+                }
+                throw throwable;
+            }
+        }
+    }
+
+    @Test
+    public void testImportEntries() throws Throwable {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        Throwable throwable = null;
+        try (InputStream is = BiblioscapeImporter.class.getResourceAsStream("BiblioscapeImporterTest.txt")) {
+            List<BibtexEntry> entries = importer.importEntries(is, new OutputPrinterToNull());
+                Assert.assertEquals(7, entries.size());
```

Comment:
How do you assert that the right seven entries with the right values are read correctly?